Modules/Sources/WordPressUI/Views/StandardForm/StandardForm.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftUI
+import DesignSystem
+
+/// A standard WordPress form with cards.
+public struct StandardForm<Content: View>: View {
```

Comment:
Not sure about the naming, but I didn't want conflicts with SwiftUI stock components.